demos/text-field.html

```diff
@@ -679,6 +731,56 @@ <h2>Full-Width Text Field and Textarea</h2>
           section.classList[target.checked ? 'add' : 'remove']('mdc-theme--dark');
         });
       });
+
+      demoReady(function() {
+
+        // Initializes all checkboxes to find closest "example" class (section), finds the elementToExecuteCallbackOn
+        // and calls a function with the event and each element found.
+        var findElementAndAddClickHandlerToCheckBox = function(checkboxClassName, elementToExecuteCallbackOn, handlerCallback) {
```

Comment:
Shorten some names here? `addCheckboxHandlers` for function name (we're not actually listening for click as the name currently suggests anyway)? `targetSelector` for second argument?